### PR TITLE
support xsi:type schema on element (bug #606)

### PR DESCRIPTION
**Pull request by pentode**

### CODE REVIEW

File: lib/wsdl.js [hunk 1]
```diff
@@ -1291,9 +1291,20 @@ WSDL.prototype.xmlToObject = function(xml) {
...
+    var _xsiTypeSchema;
+    var _xsiType = elementAttributes['xsi:type'];
+    if (_xsiType) {
+      var _type = splitNSName(_xsiType);
+      var _typeDef = self.findParameterObject(xmlns[_type.namespace], _type.name);
+      if (_typeDef) {
+        _xsiTypeSchema = _typeDef.description(self.definitions);
+      }
+    }
+
```
Comment (herom) [inline lib/wsdl.js]: Would you mind renaming your variables and remove the trailing underscore, for example `_xsiType` --> `xsiType`?  I'm aware that this is a bit picky, but in general underscores should be "reserved" for private members and methods :+1:

**Comment by pentode** [comment]
Not a problem.  I'll resubmit the PR later today.

**Comment by pentode** [comment]
@herom I've just made the requested change (and finally figure out how to correctly rebase!)

**Comment by herom** [comment]
thanks a bunch! :smiley:

**Comment by pentode** [comment]
cheers!

**Pull request merged**